run-profile

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
```

Comment:
We're not very consistent with script names on this repo.
If we look at Zeth, every script is named in `snake_case` (https://github.com/clearmatics/zeth/tree/develop/scripts). Here we have a mixture of Snake CKebab case and Kebab case. Let's follow what we do on Zeth and use Snake case for these file names.

Reply:
Scripts have been renamed in latest.

Reply:
As discussed, let's use `kebab-case` for executables and `snake_case` for non exec.

Reply:
all scripts should have been renamed in the latest commit.